src/utils/db/car_util.rs

```diff
@@ -2,10 +2,22 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use futures::{Stream, StreamExt, TryStreamExt};
-use tokio::io::{AsyncBufRead, AsyncSeek};
+use fvm_ipld_blockstore::Blockstore;
+use tokio::io::{AsyncBufRead, AsyncSeek, BufReader};
 
 use crate::cid_collections::CidHashSet;
-use crate::utils::db::car_stream::{CarBlock, CarStream};
+use crate::utils::db::car_stream::{CarBlock, CarHeader, CarStream};
+
+pub async fn load_car<R>(db: &impl Blockstore, reader: R) -> anyhow::Result<CarHeader>
```

Comment:
```suggestion
/// Stream key-value pairs from a CAR archive into a block store. The block store is not restored to its original state in case of errors.
pub async fn load_car<R>(db: &impl Blockstore, reader: R) -> anyhow::Result<CarHeader>
```